src/dodal/devices/zocalo/zocalo_results.py

```diff
@@ -106,17 +112,40 @@ def _clear_old_results(self):
         LOGGER.info("Clearing queue")
         self._raw_results_received = Queue()
 
-    def stage(self):
+    @AsyncStatus.wrap
+    async def stage(self, group=ZOCALO_STAGE_GROUP):
```

Comment:
Should: It feels a bit weird to specify the group here rather than when doing the `bps.stage` in the plan. I don't think we do it anywhere else and I feel like it makes the plan a bit less readable 'cos you're waiting on something you haven't explicitly set up. Is there a reason to do it this way that I'm missing?

Reply:
nope, and it wouldn't even work, I just got it completely wrong. thanks for catching that